test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -674,6 +674,11 @@ func RunBenchmarkPerfScheduling(b *testing.B, outOfTreePluginRegistry frameworkr
 					if !enabled(*perfSchedulingLabelFilter, append(tc.Labels, w.Labels...)...) {
 						b.Skipf("disabled by label filter %q", *perfSchedulingLabelFilter)
 					}
+					tCtx := ktesting.Init(b)
+					if !*useTestingLog {
+						// Use the normal global logger instead of the per-test logger initialized by Init.
```

Comment:
Can we move this into Init, users will not quite care about the implementation.

Reply:
No, because if we *don't* use per-test logging, then it has to be done here by reverting the default behavior of `ktesting.Init`, which is per-test logging.

Reply:
I prefer to not put a seldomly used option into the main ktesting package. Therefore I created "ktesting/initoption" and placed `PerTestOutput` there. Because it's a package specifically for functional options, there is no need to add the `With` prefix. That is used instead for the functions  which create a new context.